src/components/organisms/AppRouter/AppRouter.tsx

```diff
@@ -6,97 +6,68 @@ import {
-import { RoomsForm } from "pages/Admin/Venue/Rooms/RoomsForm";
-import { VenueAdminPage } from "pages/Admin/Venue/VenueAdminPage";
-import { AdminAdvancedSettings } from "pages/AdminAdvancedSettings";
-import { AdminVenueView } from "components/organisms/AdminVenueView";
+import { LoadingPage } from "components/molecules/LoadingPage";
+
+import { AccountSubrouter } from "./AccountSubrouter";
+import { AdminNGSubrouter } from "./AdminNGSubrouter";
+import { AdminV1Subrouter } from "./AdminV1Subrouter";
+import { EnterSubrouter } from "./EnterSubrouter";
 
 const AppRouter: React.FC = () => {
   return (
     <Router basename="/">
       <Switch>
-        <Route
-          path="/SparkleVerse"
-          component={() => <Redirect to={SPARKLEVERSE_HOMEPAGE_URL} />}
-        />
-        <Route path="/enter/step1" component={Step1} />
-        <Route path="/enter/step2" component={Step2} />
-        <Route path="/enter/step3" component={Step3} />
-        <Route path="/enter/step4" component={Step4} />
-        <Route path="/enter/step5" component={Step5} />
-        <Route path="/enter/step6a" component={Step6a} />
-        <Route path="/enter/step6b" component={Step6b} />
-        {/* @debt This appears to be a legacy route related to the old Playa template. Likely to be removed in future. */}
-        {/*<Route path="/enter" component={SplashPage} />*/}
-        <Route path="/account/profile" component={Profile} />
-        <Route path="/account/questions" component={Questions} />
-        <Route path="/account/code-of-conduct" component={CodeOfConduct} />
+        <Route path="/enter" component={EnterSubrouter} />
+        <Route path="/account" component={AccountSubrouter} />
+        <Route path="/admin" component={AdminV1Subrouter} />
+        <Route path="/admin-ng" component={AdminNGSubrouter} />
+
         <Route
           path="/login/:venueId/:customToken"
           component={LoginWithCustomToken}
         />
         {/* @debt The /login route doesn't work since we added non-defaulted props to the Login component */}
         {/*<Route path="/login" component={Login} />*/}
-        <Route path="/admin/venue/rooms/:venueId" component={RoomsForm} />
-
-        <Route path="/admin/venue/creation" component={VenueWizard} />
-        <Route path="/admin/venue/edit/:venueId" component={VenueWizard} />
-
-        <Route path="/admin-ng/venue/:venueId?" component={AdminVenueView} />
-        <Route
-          path="/admin-ng/advanced-settings/:venueId?"
-          component={AdminAdvancedSettings}
-        />
-
-        <Route path="/admin-ng/venue/creation" component={VenueWizard_v2} />
-        <Route path="/admin-ng/edit/:venueId" component={VenueWizard_v2} />
-
-        <Route path="/admin/:venueId" component={Admin} />
-
-        <Route path="/admin" component={Admin} />
-        <Route path="/admin-ng" component={Admin_v2} />
 
         <Route path="/v/:venueId" component={VenueLandingPage} />
```

Comment:
As a part of refactoring, we could also use functions from `utils/url`.
For example, for this route it would be `venueLandingUrl(':venueId')`

But probably, OOS of this PR. 🙃

Reply:
Lets call it OOS of this PR. There might be some cleaner ways for us to refactor both of those things in future, but would focus on them as a standalone PR.